sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigquery/StorageApiSinkSchemaUpdateIT.java

```diff
@@ -618,11 +619,9 @@ public void runDynamicDestinationsWithAutoSchemaUpdate(boolean useAtLeastOnce) t
     // Need to manually enable streaming engine for legacy dataflow runner
     ExperimentalOptions.addExperiment(
         p.getOptions().as(ExperimentalOptions.class), GcpOptions.STREAMING_ENGINE_EXPERIMENT);
-    // Only run the most relevant test case on Dataflow
-    if (p.getOptions().getRunner().getName().contains("DataflowRunner")) {
-      assumeTrue(
-          "Skipping in favor of more relevant test case", changeTableSchema && useInputSchema);
-    }
+    // Skipping dynamic destinations tests on Dataflow because of timing issues.
+    // These tests are more stable on the DirectRunner, where timing is less variable.
+    assumeFalse(p.getOptions().getRunner().getName().contains("DataflowRunner"));
```

Comment:
please use assumeFalse(message, ...) so from test report one could see why it is skipped

Reply:
@Abacn done

Reply:
Done